### PR TITLE
Add Vietnamese translation

### PR DESCRIPTION
**Pull request by torn4dom4n**
Hi @parksb, I closed https://github.com/parksb/handmade-blog/pull/47 and reopen this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   </h1>
 
-  <strong>Read this document in another language:</strong> [:kr:](_docs/README-KO.md) [:indonesia:](_docs/README-ID.md) [:brazil:](_docs/README-PT-BR.md) [:it:](_docs/README-IT.md) [:malaysia:](_docs/README-MS.md) [:greece:](_docs/README-EL.md)
+  <strong>Read this document in another language:</strong> [:kr:](_docs/README-KO.md) [:indonesia:](_docs/README-ID.md) [:brazil:](_docs/README-PT-BR.md) [:it:](_docs/README-IT.md) [:malaysia:](_docs/README-MS.md) [:greece:](_docs/README-EL.md) [:vietnam:](README.md)
```
Comment (parksb) [inline README.md]:
```suggestion
  <strong>Read this document in another language:</strong> [:kr:](_docs/README-KO.md) [:indonesia:](_docs/README-ID.md) [:brazil:](_docs/README-PT-BR.md) [:it:](_docs/README-IT.md) [:malaysia:](_docs/README-MS.md) [:greece:](_docs/README-EL.md) [:vietnam:](_docs/README-VI.md)
```

Please check the path

File: _docs/README-VI.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        "tableOfContents": true 
+      }
+    }
+    ```
+
+5. Khởi động máy chủ cục bộ tại `http://localhost:1234/`. Chạy lệnh `npm start` mở máy chủ cục bộ dựa trên thư mục` server`.
+
+    ```shell script
+    $ npm start
+    ```
```
Comment (parksb) [inline _docs/README-VI.md]: The contents of the fourth and fifth steps have been changed. Please check if it's translated according to the latest README.md.

```
4. Customize a some texts such as title of the navigation(in `app/templates/navigations.ejs`), and run `npm run build`.

    ```html
    <nav>
      <a class="logo-link" href="/">
        <h1>CUSTOMIZED BLOG TITLE</h1>
        <span>customized blog subtitle</span>
      </a>
      <small>
        <a id="about" class="info-link" href="/about.html">👀About</a> /
        <a id="works" class="info-link" href="/works.html">🔥Works</a> /
        <a id="articles" class="info-link" href="/articles.html">📚Articles</a>
      </small>
    </nav>
    ```
    ``` shell script
    $ npm run build
    ```

5. Run `npm start` script to start a local server listening on `http://localhost:8080/`. The local server is based on `dist` directory.

    ```shell script
    $ npm run build
    $ npm start
    ```
```

**Review by parksb — COMMENTED** [review]
Please check the comments I left. Thank you for the translation :)

**Comment by torn4dom4n** [comment]
@parksb I updated missing translations. Can you review it?

**Pull request merged**